examples/scripts/upgrade_to_ps3.ps1

```diff
@@ -20,21 +20,21 @@
 
 if ($PSVersionTable.psversion.Major -ge 3)
 {
-    write-host "Powershell 3 Installed already; You don't need this"
+    Write-Output "Powershell 3 Installed already; You don't need this"
```

Comment:
Honestly this file is mostly useless and doesn't really work properly, there's not much of a point of making changes here.